dask_kubernetes/operator/operator.py

```diff
@@ -1,16 +1,13 @@
 import asyncio
-
-from distributed.core import rpc
+import aiohttp
+import json
 
 import kopf
 import kubernetes_asyncio as kubernetes
 
 from uuid import uuid4
 
 from dask_kubernetes.common.auth import ClusterAuth
-from dask_kubernetes.common.networking import (
-    get_scheduler_address,
```

Comment:
We probably still need to do this as this handles accessing the scheduler in a variety of different service modes. Although we don't want the comm anymore we want the HTTP server (port `8787`). 

Luckily @mmccarty has taken us part of the way down this path by making the port forwarding code more configurable in #489.

We probably still need to expand `get_scheduler_address` and `get_external_address_for_scheduler_service` in `dask_kubernetes.common.networking` to also accept a kwarg to choose which port we want to forward. Right now we assume the port is called `"comm"` but this should be configurable as we would need to set it to `"dashboard"` here.

https://github.com/dask/dask-kubernetes/blob/feedbf40b974774c53f73e02cb3ea3fd72192c14/dask_kubernetes/common/networking.py#L20